src/index.ts

```diff
@@ -118,7 +118,7 @@ export const hashLimitOrder = (limitOrder: LimitOrder) => {
   return getLimitOrderMsgHash(...args);
 };
 
-export const sign = (privateKey: string, message: string) => {
+const sign = (privateKey: string, message: string) => {
```

Comment:
Here it's not inferred and we could make it explicit. I think it's the only occurrence in this file 
```suggestion
const sign = (privateKey: string, message: string): Signature => {
```